gateway-modules/apim-ballerina-configs/configs/org/wso2/carbon/apimgt/gateway/utils/APICoreUtil.bal

```diff
@@ -89,7 +89,7 @@ function getAPIServiceConfig (string apiId) (string) {
 
 function getAPICoreURL () (string){
     string apiCoreURL = system:getEnv(Constants:API_CORE_URL);
-    return apiCoreURL;
+    return "https://localhost:9292";
```

Comment:
Why do we need to hard code this? we should be able to generate the URL using apiCoreURL read from an environment variable in the previous line.